lang/en.js

```diff
@@ -230,7 +230,7 @@
     "constants.plans.pro": "Pro plan",
     "constants.plans.biz": "Business plan",
     "constants.plans.ent": "Enterprise plan",
-    "errors.noActiveZoneSelected": "Please select a domain that is provisioned with Cloudflare.",
+    "errors.noActiveZoneSelected": "Please continue to {link} to set up this domain.",
```

Comment:
And we need a conditional for cPanel which links them to the domains page since those domains are provisoned in app.